app/views/solidus_friendly_promotions/admin/shared/_promotion_sub_menu.html.erb

```diff
@@ -0,0 +1,8 @@
+<ul class="admin-subnav" data-hook="admin_promotion_sub_tabs">
+  <% if can? :admin, SolidusFriendlyPromotions::Promotion %>
```

Comment:
Do these need to be separate permission sets from Spree::Promotion? The upgrade/migration path is simpler if they are not.

Reply:
The permission sets for this namespace do not exist yet.